Gemfile

```diff
@@ -17,20 +17,18 @@ gem 'paperclip', '4.1.1'
 gem 'progress', require: false
 gem 'pry-rails'
 gem 'rack-cors', require: 'rack/cors'
-gem 'rails', '4.2.11.1'
+gem 'rails', '5.0.7.2'
 gem 'rake', '10.0.3'
 gem 'rb-gsl', '1.16.0.6'
 gem 'rubyzip', '>= 1.0.0'
 gem 'sidekiq'
 gem 'sidekiq-unique-jobs', '3.0.12'
-gem 'sinatra', require: false
```

Comment:
It seems like it was installed when adding the Sidekiq web panel. But the Sidekiq docs don't require that